test/LockTest.js

```diff
@@ -1,74 +1,232 @@
-            assert.isTrue(hasInitialized)
+          const actualLockCount = await lockForwarder.getWithdrawLocksCount(rootAccount)
+          assert.equal(actualLockCount, expectedLockCount)
         })
 
-        describe('changeLockDuration(uint256 _lockDuration)', () => {
+        it("can't withdraw more than locked", async () => {
+          await assertRevert(lockForwarder.withdrawTokens(lockCount + 1), 'LOCK_TOO_MANY_WITHDRAW_LOCKS')
+        })
 
-            it('sets a new lock duration', async () => {
-                await daoDeployment.acl.createPermission(rootAccount, lockForwarder.address, CHANGE_DURATION_ROLE, rootAccount)
-                const expectedLockDuration = 120
+        it('withdraws 1 locked token', async () => {
+          const locksToWithdraw = 1
+          const addressPrevBalance = await mockErc20.balanceOf(rootAccount)
 
-                await lockForwarder.changeLockDuration(expectedLockDuration)
+          const expectedLockCount = lockCount - locksToWithdraw
+          const expectedBalance = addressPrevBalance.toNumber() + locksToWithdraw * INITIAL_LOCK_AMOUNT
 
-                const actualLockDuration = await lockForwarder.lockDuration()
-                assert.strictEqual(actualLockDuration.toNumber(), expectedLockDuration)
-            })
+          //increase time
+          await lockForwarder.mockIncreaseTime(INITIAL_LOCK_DURATION + 1)
+          await lockForwarder.withdrawTokens(locksToWithdraw, { from: rootAccount })
+
+          const actualLockCount = await lockForwarder.getWithdrawLocksCount(rootAccount)
+          const actualBalance = await mockErc20.balanceOf(rootAccount)
+          assert.equal(actualLockCount, expectedLockCount)
+          assert.equal(actualBalance, expectedBalance)
         })
 
-        describe('changeLockAmount(uint256 _lockAmount)', () => {
+        //Having issue when calling withdrawTokens() (Invalid number of arguments in Solidity function)
+        it(`withdraws all locked tokens (${lockCount})`, async () => {
+          const expectedLockCount = 0
 
-            it('sets a new lock amount', async () => {
-                await daoDeployment.acl.createPermission(rootAccount, lockForwarder.address, CHANGE_AMOUNT_ROLE, rootAccount)
-                const expectedLockAmount = 20
+          //increase time
+          await lockForwarder.mockIncreaseTime(INITIAL_LOCK_DURATION + 1)
+          await lockForwarder.withdrawTokens(lockCount, { from: rootAccount })
 
-                await lockForwarder.changeLockAmount(expectedLockAmount)
+          const actualLockCount = await lockForwarder.getWithdrawLocksCount(rootAccount)
+          assert.equal(actualLockCount, expectedLockCount)
+        })
 
-                const actualLockAmount = await lockForwarder.lockAmount()
-                assert.strictEqual(actualLockAmount.toNumber(), expectedLockAmount)
-            })
+        describe('Change configuration parameters', async () => {
```

Comment:
For consistency I would create 2 describes, one for each of 'changeLockDuration(uint256 _lockDuration)' and 'changeLockAmount(uint256 _lockAmount)' and have the tests related to those functions within them. This makes it clear what's being tested in the output then, like the previous tests.